services/compute/virtualmachine/virtualmachine.go

```diff
@@ -58,6 +58,11 @@ func (c *client) getWssdVirtualMachine(vm *compute.VirtualMachine, group string)
 		return nil, errors.Wrapf(err, "Failed to get AvailabilityZone Profile")
 	}
 
+	placementGroupProfile, err := c.getWssdPlacementGroupReference(vm.PlacementGroupProfile)
+	if err != nil {
```

Comment:
error will never be thrown here as inside `getWssdPlacementGroupReference`, you are not throwing any error.

Reply:
removed dead code in new patch